### ISSUE TITLE
How to clear queue or send same file again

### ISSUE THREAD

**Issue by JanneHarju** [jkuri/ngx-uploader#287]
If user select one file and uploads it and do same again to same file nothing happend.

**Comment by dmeneses** [comment]
Reproducible only on chrome it seems, not in firefox.

**Comment by yonimoses** [comment]
Hi
Any updated or solution for this? I'm facing the same issue.

**Comment by retailify** [comment]
@yonimoses do you work with 4.2.1?

**Comment by Vishal-Isharani** [comment]
i'm also facing this issue and i'm using @4.2.1 but i think it's chrome only issue

**Comment by yonimoses** [comment]
@retailify Yes, with 4.2.1 . angular 5.1.1 (can't update ng version cause of the AOT compile issues). I tried to reset the file input using JS, didn't work as well.
